templates/statefulset.yaml

```diff
@@ -130,11 +143,20 @@ spec:
           prop_replace nifi.web.http.host ${FQDN}
           prop_replace nifi.zookeeper.connect.string ${NIFI_ZOOKEEPER_CONNECT_STRING}
 
+          prop_replace nifi.security.keystorePasswd ${KEYSTORE_PASSWORD}
```

Comment:
We should probably figure out how to add these secrets using a volume per the chart implementation, since we don't want to check this in - https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/#create-a-pod-that-has-access-to-the-secret-data-through-a-volume